cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesModel.java

```diff
@@ -166,6 +166,8 @@ static String baseName(ReadOnlyDataSource ds) {
 
     void write(DataSource ds);
 
+    void write(DataSource ds, CgmesSubset subset);
```

Comment:
Not sure it is used anywhere else but it may be safer to add a default implementation